ote/src/cljs/ote/views/parking.cljs

```diff
@@ -1,51 +1,195 @@
 (ns ote.views.parking
-  "Todo"
-  (:require [reagent.core :as reagent]
+  "Required data input fields for parking services"
+  (:require [reagent.core :as r]
             [cljs-react-material-ui.reagent :as ui]
             [cljs-react-material-ui.icons :as ic]
             [ote.ui.form :as form]
             [ote.ui.form-groups :as form-groups]
             [ote.ui.buttons :as buttons]
-            [ote.app.controller.transport-service :as ts]
             [ote.app.controller.parking :as parking]
-            [ote.db.transport-service :as transport-service]
+            [ote.db.transport-service :as t-service]
             [ote.db.common :as common]
             [ote.localization :refer [tr tr-key]]
             [ote.views.place-search :as place-search]
-            [tuck.core :as tuck]))
+            [tuck.core :as tuck]
+            [stylefy.core :as stylefy]
+            [ote.style.base :as style-base]
+            [ote.style.form :as style-form]
+            [ote.app.controller.transport-service :as ts]
+            [ote.views.transport-service-common :as ts-common]
+            [ote.time :as time]))
 
-(defn parking-form-options [e!]
-  {:name->label (tr-key [:field-labels :terminal])
-   :update!     #(e! (parking/->EditParkingState %))
+(defn form-options [e!]
+  {:name->label (tr-key [:field-labels :parking]
+                        [:field-labels :transport-service-common]
+                        [:field-labels :transport-service]
+                        [:field-labels])
+   :update!     #(e! (ts/->EditTransportService %))
    :name        #(tr [:olennaiset-tiedot :otsikot %])
    :footer-fn   (fn [data]
-                  [buttons/save {:on-click #(e! (parking/->SaveParkingToDb))
-                                   :disabled (form/disable-save? data)}
-                   (tr [:buttons :save])])})
-
-(defn parking [e! status]
-  [:div.row
-   [:div {:class "col-lg-12"}
-    [:div
-     [:h1 "Täydennä pysäköintiin liittyvät tiedot."]]
-
-    [:div {:style {:border "dotted 5px red"
-                   :padding "2em"
-                   :margin "1em"}}
-     "Tämä lomake on työn alla! Tulossa testikäyttöön pian!"]
-
-    [form/form (parking-form-options e!)
-
-     [
-      (form/group
-        {:label   "Pakkipaikka - tai jotain"
-         :columns 1}
-        {:name        ::transport-service/:eligibility-requirements
-         :type        :localized-text
-         }
-        )
-      ]
-
-     (get status ::transport-service/parking)]
-
-    ]])
+                  [ts-common/footer e! data])})
+
+(defn name-and-type-group [e!]
```

Comment:
Name could be grouped with contact-details or maybe with some other data set.
It is kind of lonely here :)

Reply:
Hmm, most of the contant-info groups are from ts-common. Harder to inject the name group there. I'd rather use the common info groups than implement a custom one at this point.